client/src/app/components/page/list/list.component.ts

```diff
@@ -34,7 +34,14 @@ export class ListComponent implements OnInit, OnDestroy {
             takeUntil(this.ngUnsubscribe)
         ).subscribe(
             tasks => {
-                this.tasks = tasks;
+                this.tasks = tasks.map(task=>{
+                    task = Object.assign(task, {
+                        createdAt:new Date(task.createdAt),
+                        updatedAt:new Date(task.updatedAt)
+                    });
```

Comment:
This form is ok, but there is ES6 spread operator, which is much more convinient:

`task = { ...task, createdAt: new Date(task.createdAt), updatedAt: new Date(task.updatedAt) }`

Reply:
Also deleted because of date piping.